src/Umbraco.Web/PublishedContentQuery.cs

```diff
@@ -70,6 +79,13 @@ public IEnumerable<IPublishedContent> TypedContent(IEnumerable<int> ids)
                 : _typedContentQuery.TypedContent(ids);
         }
 
+        public IEnumerable<IPublishedContent> TypedContent(IEnumerable<Guid> ids)

```

Comment:
Code looks/reads fine so far btw @zpqrtbnk just nitpicking to ensure we try to get into a better of habbit of documenting even if it's obvious to us internally. It's not always obvious to developers.